repos/fdbt-site/src/pages/salesConfirmation.tsx

```diff
@@ -37,6 +57,9 @@ export const buildSalesConfirmationElements = (
     salesOfferPackages: SalesOfferPackage[] | ProductWithSalesOfferPackages[],
     startDateIn: string,
     endDateIn: string | null,
+    fareType: string,
+    caps: CapInfo[],
+    fullCaps?: CapsDefinition | CapsDefinitionWithErrors,
```

Comment:
It has just occurred to me that we're passing through an array of caps, and the cap they selected. Instead we just need to have selectedCap?: CapsDefinition as the prop, and caps: CapInfo[] is not needed.

Then on line 112, we need to do 

`if (['single', 'return', 'flatFare'].includes(fareType) && !!selectedCap) {`